src/support_sphere/lib/presentation/components/auth/signup_form.dart

```diff
@@ -180,10 +217,20 @@ class _ConfirmedPasswordInput extends StatelessWidget {
           onChanged: (password) =>
               context.read<SignupCubit>().confirmedPasswordChanged(password),
           obscureText: !state.showPassword,
+          autovalidateMode: AutovalidateMode.onUserInteraction,
+          validator: (value) => validateValue(
+            [
+              FormBuilderValidators.required(),
+              FormBuilderValidators.password(),
+              (String? val) =>
+                  val != state.password ? 'Passwords do not match' : null,
+            ],
```

Comment:
I think it's enough for the confirm password input to just check if the passwords match -- if they do, they meet all the password requirements already :)